packages/performance-tests/test/TransformerRegression.test.ts

```diff
@@ -157,12 +176,18 @@ async function runRegressionTests() {
         });
 
         testCasesMap.forEach(async (testCase, key) => {
-          it(key, async () => {
-            await testCase(sourceDb,
-              (...smallReportSubset: [testName: string, iModelName: string, valDescription: string, value: number]) => {
-                reporter.addEntry( ...smallReportSubset, reportInfo );
+          transformerModules.forEach((transformerModule: TestTransformerModule, moduleName: string) => {
+            it(`${key} on ${moduleName}`, async () => {
+              await testCase({
+                sourceDb,
+                transformerModule,
+                addReport: (...smallReportSubset: [testName: string, iModelName: string, valDescription: string, value: number]) => {
```

Comment:
if `testCase` is properly typed, you shouldn't need to explicitly specify the type of the rest argument